2025-02-09 19:18:03,605 INFO    MainThread:34880 [wandb_init.py:_log_setup():523] Logging user logs to c:\Users\Lenovo\Documents\Projects\Kaggle\Natural Language Processing With Disaster Tweets\wandb\run-20250209_191803-ztnwxlel\logs\debug.log
2025-02-09 19:18:03,606 INFO    MainThread:34880 [wandb_init.py:_log_setup():524] Logging internal logs to c:\Users\Lenovo\Documents\Projects\Kaggle\Natural Language Processing With Disaster Tweets\wandb\run-20250209_191803-ztnwxlel\logs\debug-internal.log
2025-02-09 19:18:03,606 INFO    MainThread:34880 [wandb_init.py:_jupyter_setup():469] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000126601B3380>
2025-02-09 19:18:03,606 INFO    MainThread:34880 [wandb_init.py:init():641] calling init triggers
2025-02-09 19:18:03,606 INFO    MainThread:34880 [wandb_init.py:init():647] wandb.init called with sweep_config: {}
config: {'model': 'Conv1D', 'epochs': 5, 'loss_function': 'binary_crossentropy', 'optimizer': 'Adam', 'metrics': 'accuracy'}
2025-02-09 19:18:03,606 INFO    MainThread:34880 [wandb_init.py:init():674] starting backend
2025-02-09 19:18:03,606 INFO    MainThread:34880 [wandb_init.py:init():678] sending inform_init request
2025-02-09 19:18:03,609 INFO    MainThread:34880 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2025-02-09 19:18:03,609 INFO    MainThread:34880 [wandb_init.py:init():693] backend started and connected
2025-02-09 19:18:03,611 INFO    MainThread:34880 [wandb_run.py:_label_probe_notebook():1226] probe notebook
2025-02-09 19:18:03,634 INFO    MainThread:34880 [wandb_run.py:_label_probe_notebook():1236] Unable to probe notebook: 'charmap' codec can't decode byte 0x8f in position 8612: character maps to <undefined>
2025-02-09 19:18:03,637 INFO    MainThread:34880 [wandb_init.py:init():786] updated telemetry
2025-02-09 19:18:04,177 INFO    MainThread:34880 [wandb_init.py:init():818] communicating run to backend with 90.0 second timeout
2025-02-09 19:18:04,672 INFO    MainThread:34880 [wandb_init.py:init():868] starting run threads in backend
2025-02-09 19:18:04,865 INFO    MainThread:34880 [wandb_run.py:_console_start():2415] atexit reg
2025-02-09 19:18:04,865 INFO    MainThread:34880 [wandb_run.py:_redirect():2265] redirect: wrap_raw
2025-02-09 19:18:04,865 INFO    MainThread:34880 [wandb_run.py:_redirect():2330] Wrapping output streams.
2025-02-09 19:18:04,865 INFO    MainThread:34880 [wandb_run.py:_redirect():2355] Redirects installed.
2025-02-09 19:18:04,866 INFO    MainThread:34880 [wandb_init.py:init():910] run started, returning control to user process
2025-02-09 19:18:04,869 INFO    MainThread:34880 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-09 19:18:04,870 INFO    MainThread:34880 [wandb_init.py:_pause_backend():434] pausing backend
2025-02-09 19:18:05,642 INFO    MainThread:34880 [wandb_init.py:_resume_backend():439] resuming backend
2025-02-09 19:18:16,068 INFO    MainThread:34880 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-09 19:18:16,068 INFO    MainThread:34880 [wandb_init.py:_pause_backend():434] pausing backend
2025-02-09 19:18:16,556 INFO    MainThread:34880 [wandb_init.py:_resume_backend():439] resuming backend
2025-02-09 19:18:16,907 INFO    MainThread:34880 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-09 19:18:16,907 INFO    MainThread:34880 [wandb_init.py:_pause_backend():434] pausing backend
2025-02-09 19:18:16,911 INFO    MainThread:34880 [wandb_init.py:_resume_backend():439] resuming backend
2025-02-09 19:18:16,915 INFO    MainThread:34880 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-09 19:18:16,915 INFO    MainThread:34880 [wandb_init.py:_pause_backend():434] pausing backend
2025-02-09 19:18:16,922 INFO    MainThread:34880 [wandb_init.py:_resume_backend():439] resuming backend
2025-02-09 19:18:16,928 INFO    MainThread:34880 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-09 19:18:16,929 INFO    MainThread:34880 [wandb_init.py:_pause_backend():434] pausing backend
2025-02-09 19:18:16,936 INFO    MainThread:34880 [wandb_init.py:_resume_backend():439] resuming backend
2025-02-09 19:18:16,940 INFO    MainThread:34880 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-09 19:18:16,942 INFO    MainThread:34880 [wandb_init.py:_pause_backend():434] pausing backend
2025-02-09 19:18:17,069 INFO    MainThread:34880 [wandb_init.py:_resume_backend():439] resuming backend
2025-02-09 19:18:17,073 INFO    MainThread:34880 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-09 19:18:17,073 INFO    MainThread:34880 [wandb_init.py:_pause_backend():434] pausing backend
2025-02-09 19:18:17,377 INFO    MainThread:34880 [wandb_init.py:_resume_backend():439] resuming backend
2025-02-09 19:18:17,382 INFO    MainThread:34880 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-09 19:18:17,382 INFO    MainThread:34880 [wandb_init.py:_pause_backend():434] pausing backend
2025-02-09 19:18:19,284 INFO    MainThread:34880 [wandb_init.py:_resume_backend():439] resuming backend
2025-02-09 19:18:19,286 INFO    MainThread:34880 [wandb_run.py:_finish():2140] finishing run althariqfairuz273-institut-teknologi-bandung/nlp-disaster-tweets/ztnwxlel
2025-02-09 19:18:19,286 ERROR   MainThread:34880 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-02-09 19:18:19,286 INFO    MainThread:34880 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-09 19:18:19,286 INFO    MainThread:34880 [wandb_init.py:_jupyter_teardown():451] cleaning up jupyter logic
2025-02-09 19:18:19,286 INFO    MainThread:34880 [wandb_run.py:_atexit_cleanup():2380] got exitcode: 0
2025-02-09 19:18:19,287 INFO    MainThread:34880 [wandb_run.py:_restore():2362] restore
2025-02-09 19:18:19,287 INFO    MainThread:34880 [wandb_run.py:_restore():2368] restore done
2025-02-09 19:18:21,535 INFO    MainThread:34880 [wandb_run.py:_footer_history_summary_info():3984] rendering history
2025-02-09 19:18:21,535 INFO    MainThread:34880 [wandb_run.py:_footer_history_summary_info():4016] rendering summary
2025-02-09 19:18:21,545 INFO    MainThread:34880 [wandb_run.py:_footer_sync_info():3945] logging synced files
